Client/Frontend/Fakespot/FakespotViewModel.swift

```diff
@@ -13,107 +13,113 @@ class FakespotViewModel {
         case loaded(ProductAnalysisData?, AnalysisStatus?, analysisCount: Int)
         case error(Error)
 
-        fileprivate var viewElements: [ViewElement] {
+        fileprivate var productData: ProductAnalysisData? {
             switch self {
-            case .loading:
-                return [.loadingView]
-            case let .loaded(product, analysisStatus, analysisCount):
-                guard let product else {
-                    return [
-                        .messageCard(.genericError),
-                        .qualityDeterminationCard,
-                        .settingsCard
-                    ]
-                }
+            case .loading, .error, .onboarding: return nil
+            case .loaded(let data, _, _): return data
+            }
+        }
+    }
+
+    fileprivate func viewElements(for viewState: ViewState) -> [ViewElement] {
```

Comment:
I moved `viewElements` to be a function on view model to be able to use feature flags.